manual/2-populate-the-structure/templates/reference/reference-template.md

```diff
@@ -1,212 +1,164 @@
-- For content, use the same format as H2 sections: paragraph, list, or table.
-- Stop at H4. Deeper levels (H5, H6) are forbidden.
--->
-
-(Optional) #### { H4 section content }
-
-{ Paragraph, list, or table content }
-
-(Optional) { Screenshot }
-
-(Optional) { Code }
+> Keep container resources within the documented limits to avoid throttling.
+
+## Intro <!-- group: REF-INTRO -->
+
+Every reference requires a short introduction that provides context.
+
+- Write one or two 50- to 100-word paragraphs explaining the purpose and relevance of the reference article. <!-- REF-BEHAV-INTRO-LENGTH-50-100 -->
+- Provide context or background information to help readers understand the information. <!-- REF-BEHAV-INTRO-CONTEXT -->
+- Link only when it helps disambiguate or deep-dive (aim ≤2 links). <!-- REF-BEHAV-INTRO-LINK-HEADERS -->
+- If the intro is long or complex, link to other supportive documents. <!-- REF-BEHAV-INTRO-LINK-DOCS -->
+- Avoid meta-descriptions, such as `The following table [...]`, `This article includes the list of [...]`, or similar. <!-- REF-BEHAV-INTRO-OMIT-META -->
+- This section is guidance only; do not render a visible heading or body. <!-- REF-STRUCT-INTRO-GUIDELINES-NO-RENDER -->
+
+## H2 section <!-- group:REF-H2-SECTION -->
+
+- Break down long lists or tables into smaller, more manageable sections using H2 headings. <!-- REF-BEHAV-H2-SECTION-BREAKDOWN -->
+	- Examples: Use different lists or tables for different categories, like `Configuration`, `Authentication`, or `Networking`.
+- One heading = one concept. Don't mix two concepts under the same heading. <!-- REF-BEHAV-H2-SECTION-ONE-IDEA -->
+- Arrange H2 sections from general to specific, or from most important to least important. <!-- REF-BEHAV-H2-SECTION-ORDER -->
+- Start with a paragraph before you add lists or tables. <!-- REF-STRUCT-H2-SECTION-PARAGRAPH-FIRST -->
+- Refer to this list of common titles for H2 and H3 headings: <!-- REF-BEHAV-H2-H3-SECTIONS-COMMON-TITLES -->
+
+	| Typical content | Heading text (choose one) |
+	|:---|:---|
+	| Brief definition + when/why to use it | `Overview` / `Description` / `Purpose` |
+	| Command or short config example with required/optional parts | `Usage` / `Synopsis` / `Syntax` |
+	| Options/parameters with purpose, accepted values, defaults | `Options` / `Flags` / `Parameters` / `Arguments` |
+	| Runnable examples of common tasks | `Examples` / `Example usage` |
+	| Log lines or exit codes and their meaning | `Return values` / `Output` / `Exit codes` |
+	| Typical errors, causes, and concise fixes | `Errors` / `Troubleshooting` |
+
+### Tables, lists, or paragraphs <!-- group: REF-H2-SECTION-CONTENT -->
+
+- Use tables or lists primarily to present reference data. <!-- REF-BEHAV-H2-SECTION-TABLES-LISTS -->
+- If present, screenshots have non-empty alt text. <!-- REF-STRUCT-H2-SHOT-ALT-TEXT -->
+- Follow the screenshot formatting rules in the Writing Rules. <!-- REF-BEHAV-H2-SHOT-REFER-WRITING-RULES -->
```

Comment:
Do we want to capitalize the first letter of "Writing Rules"? I've also seen occrences of "writing rules" and "Writing rules"